### PR TITLE
[ipinfo] isTimeSensitive 

### PR DESCRIPTION
**Pull request by shmuel44**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link](https://jira-dc.paloaltonetworks.com/browse/CIAC-10356) to the issue

## Description
Added support for `isTimeSensitive` feature.
when the `demisto.isTimeSensitive` is `true` we reduce the timeout to 2 seconds,

### CODE REVIEW

File: Packs/ipinfo/ReleaseNotes/2_1_22.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### IPinfo v2
+
+Updated the ***ip*** command when used for auto enrichment, the command timeout is reducing to 2 seconds.
```
Comment (dorschw) [inline Packs/ipinfo/ReleaseNotes/2_1_22.md:6]: Let's discuss it with a technical writer

**Review by dorschw — COMMENTED** [review]
Nice job!

File: Packs/ipinfo/ReleaseNotes/2_1_22.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### IPinfo v2
+
+Updated the ***ip*** command when used for auto enrichment with auto-extract=inline, the command timeout is reducing to 2 seconds.
```
Comment (dorschw) [inline Packs/ipinfo/ReleaseNotes/2_1_22.md:6]:
```suggestion
Updated the ***ip*** command, reducing the timeout of 2 seconds when used for auto enrichment using `auto-extract=inline`.
```

**Comment by ShirleyDenkberg** [comment]
@dorschw Doc review completed.

File: Packs/ipinfo/ReleaseNotes/2_1_22.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### IPinfo v2
+
+Updated the ***ip*** command, reducing the timeout of 2 seconds when used for auto enrichment using `auto-extract=inline`.
```
Comment (ShirleyDenkberg) [inline Packs/ipinfo/ReleaseNotes/2_1_22.md:6]:
```suggestion
Updated the ***ip*** command, reducing the timeout from 20 seconds to 2 seconds when used for auto enrichment using `auto-extract=inline`.
```

File: Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2.py [hunk 4]
```diff
@@ -200,6 +197,7 @@ def main() -> None:
     reliability = params.get('integrationReliability')
 
     demisto.debug(f'Command being called is {command}')
+    demisto.debug(f'Is time sensitive: {is_time_sensitive()}')
```
Comment (dorschw) [inline Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2.py:200]: let's call `is_time_sensitive` only once (server call), then pass its result to the Client as an attribute

**Review by dorschw — APPROVED** [review]

**Pull request merged**